apps/sms/views/shared/js/utils.js

```diff
@@ -711,7 +711,7 @@
        * flow is paused until yielded Promise is resolved, so that consumer gets
        * Promise result instead of Promise instance itself.
        * See https://www.promisejs.org/generators/ as the reference.
-       * @param {function*} generatorFunction Generator function that yields
+       * @param {function(...*): Iterator} generatorFunction Generator function that yields
```

Comment:
LINT error: apps/sms/views/shared/js/utils.js: line 830, col 91, Line is too long. (ERROR)